src/util.rs

```diff
@@ -200,7 +201,7 @@ pub fn default_error_handler(scene: &mut Scene, node: &usvg::Node) {
     scene.fill(
         Fill::NonZero,
         Affine::IDENTITY,
-        Color::RED.multiply_alpha(0.5),
+        color::palette::css::RED.multiply_alpha(0.5),
```

Comment:
This can just be `palette::css::RED.with_alpha(0.5)` (no `color::` prefix, and `with_alpha` instead of `multiply_alpha`).